### DISCUSSION TITLE
Custom "email subject" for email templates using Ory Cloud

### DISCUSSION THREAD

**Discussion by binoiii** [ory/kratos#3690] [Q&A] [answered]
Hi team,

We are using Ory Cloud and our use case (for invite flow, related to this [discussion](https://github.com/ory/kratos/discussions/3688)) requires us to have a custom email subject depending on the identity traits.

I would like just to double check if it is possible?

As per further research [[reference]](https://www.ory.sh/docs/kratos/emails-sms/custom-email-templates#creating-templates)., it seems that it is only possible for self hosted Ory instances via config.

Thanks in advance!

**Comment by vinckr** [comment, marked as answer]
Hello @binoiii 

Please reach out to support@ory.sh to get help for your Ory Network project!
